### PR TITLE
Release/v0.1

### PR DESCRIPTION
**Pull request by oskarth**
Closes https://github.com/status-im/nim-waku/issues/286

This is the initial beta release. Also adds some docs.

### CODE REVIEW

**Comment by oskarth** [comment]
Does this look reasonable to y'all?

Also see https://github.com/status-im/nim-waku/releases/tag/v0.1

**Review by jm-clius — APPROVED** [review]
Yup, makes sense!

File: docs/contributors/release-process.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+For more context, see https://trunkbaseddevelopment.com/branch-for-release/
+
+## How to to do releases
+
+1. Checkout a release branch from master
+
+`git checkout -b release/v0.1`
+
+2. Update CHANGELOG and ensure it is up to date
```
Comment (staheri14) [inline docs/contributors/release-process.md:13]: CHANGELOG.md file, right?

**Review by staheri14 — APPROVED** [review]
Looks good! 👍

File: docs/contributors/release-process.md (hunk 1 above)
Comment (oskarth) [reply docs/contributors/release-process.md:13]: Nice catch, fixed!

**Pull request merged**